src/rviz/visualization_manager.cpp

```diff
@@ -518,6 +524,11 @@ void VisualizationManager::updateFps()
   }
 }
 
+void VisualizationManager::updateHeadlight()
```

Comment:
To make it clearer that this is a toggle, it would be appropriate to name the property `default_light_enabled_property_` and rename the function to `updateDefaultLightVisible`, similar to: https://github.com/ros-visualization/rviz/blob/21a76fa898e66ad65a622fd982e47aec67cfc1c7/src/rviz/default_plugin/robot_model_display.cpp#L133